core/modules/wiki.js

```diff
@@ -1064,6 +1069,21 @@ exports.search = function(text,options) {
 		} else {
 			searchTermsRegExps = [new RegExp("(" + $tw.utils.escapeRegExp(text) + ")",flags)];
 		}
+	} else if(options.whitespace) {
+		terms = [];
+		$tw.utils.each(text.split(/\s+/g),function(term) {
+			if(term) {
+				terms.push($tw.utils.escapeRegExp(term));
+			}
+		});
+		searchTermsRegExps = [new RegExp("(" + terms.join("\\s+") + ")",flags)];
+	} else if(options.regexp) {
+		try {
+			searchTermsRegExps = [new RegExp("(" + text + ")",flags)];			
+		} catch(e) {
+			searchTermsRegExps = null;
+console.log("Regexp error",e)
```

Comment:
???

Reply:
It's not clear what we should do with a regexp error. Dumping to the console gives the user a chance to discover that there's something wrong. I'll improve the message to include the actual regexp.